### DISCUSSION TITLE
Application dawnload

### DISCUSSION THREAD

**Discussion by TALUKDAR887** [SEVENNetworks/AdClear-Content-Blocker-Open-Source-Issue-Tracker#4] [Q&A] [answered]
Adclear app dawnload

**Comment by TALUKDAR887** [comment]
Support

**Comment by TALUKDAR887** [comment]
Not dawnload app

**Comment by DustinMorgan** [comment, marked as answer]
Hey @TALUKDAR887 sorry we missed this one. AdClear Content Blocker is available to download through either the Galaxy Store or Google Play.
